package.json

```diff
@@ -41,7 +40,7 @@
     "deploy:ci": "gatsby build && now ./public -A ../now.json -e NODE_ENV=production --token $NOW_TOKEN --team=prideinlondon && now alias --token $NOW_TOKEN --team=prideinlondon"
   },
   "lint-staged": {
-    "src/**/*.{js,json}": [
+    "src/**/*.{js,json,css,md}": [
```

Comment:
Had an issue with this previously so I had to remove CSS from here. If u make any change to the css file and try to commit it throws an unexpected token error. Doesn't matter whats in the CSS file - if you delete everything and just have 1 line of css it will still error.

Reply:
hmm I think this was a mistake during a rebase. I'll revert it.